packages/blitz/src/cli/index.ts

```diff
@@ -36,23 +35,22 @@ const commands: {[command: string]: () => Promise<CliCommand>} = {
   codegen: () => import("./commands/codegen").then((i) => i.codegen),
   db: () => import("./commands/db").then((i) => i.db),
 }
-const foundCommand = Boolean(commands[args._[0] as string])
+
+const aliases: typeof commands = {
+  d: commands.dev!,
+  b: commands.build!,
+  s: commands.start!,
+  n: commands.new!,
+  g: commands.generate!,
+}
```

Comment:
We need to improve this a little — `aliases` is not a type of `commands`, because it has different keys. What we want instead is to say that `aliases` is an object with custom keys and values from the `commands` object. 

In this case, we don't have to write anything because the type will be inferred and TS will know the proper types.  In many cases — the fewer types the better. What's more, I'm also going to push PR to main that removes the type of `const commands` — we can let TS infer the types for us. We do it because sometimes if we type "too much", we accidentaly hide/override what TS would infer.

Reply:
(I took the liberty to push this change when resolving conflicts)